.github/workflows/test.yml

```diff
@@ -43,3 +43,12 @@ jobs:
       - name: Run tests
         run: |
           make test
+  collector:
```

Comment:
Is this needed so you can select it as a required check in branch protection? If so can you add a comment to that affect.

I often find this pattern a little confusing when looking at just the action yaml.